dangerfile.js

```diff
@@ -86,11 +86,15 @@ const noFlowFiles = newJsFiles
 
 raiseIssueAboutPaths(warn, noFlowFiles, '@flow');
 
+// based on assumptions from https://github.com/nodejs/node/wiki/ES6-Module-Detection-in-Node
+// basically it needs to contain at least one line with import or export at the
+// beginning of it, followed by a space; any content thereafter doesn't matter
+const esModuleRegex = /^(import|export)\s/g;
```

Comment:
I think it doesn't, it's a Flow convenience syntax that is being stripped by babel transform so node module resolver shouldn't be aware of it anyway, but I'm not 100% sure. @cpojer?